adlfs/tests/test_spec.py

```diff
@@ -2045,3 +2046,82 @@ def test_open_file_x(storage: azure.storage.blob.BlobServiceClient, tmpdir):
         with fs.open("data/afile", "xb") as f:
             pass
     assert fs.cat_file("data/afile") == b"data"
+
+
+def test_number_of_blocks(storage, mocker):
```

Comment:
Let's update this test name to be `test_uses_block_size_for_partitioned_uploads` in order to be a bit more self-descriptive of what we are trying to test.